CHANGELOG.rst

```diff
@@ -2,14 +2,141 @@
 Changelog
 =========
 
+2.7.0-alpha.0 (2024-01-05)
+==========================
+
+This is an alpha release, meaning it is not finished yet or suitable for production use.
+
+**New features**
+
+Improved backend validation robustness, mainly by validating new components:
+
+* [#4035] Added an E2E test for the file component.
+* [#72] Added validation for the following components: time, selectboxes, textarea, postcode, bsn, select, checkbox,
+  currency, signature, map, cosign, password, iban and licenseplate.
+
+Other features:
+
+* [#4031] Added a confirm popup when switching to v2 options.
+* [#4041] Improved API documents registration robustness, taking into account possible crashes when sending the documents.
+* [#3969] Removed the ability to override LoA for eHekrenning/eIDAS.
```

Comment:
Probably more needs to be described for this one

Reply:
Yep, something like:

> For eHerkenning/eIDAS authentication, the level of assurance can no longer be overridden. Brokers do not support this.